contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -508,35 +489,37 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, ERC677Recei
   /**
    * @inheritdoc IFunctionsSubscriptions
    */
-  function timeoutRequests(bytes32[] calldata requestIdsToTimeout) external override {
+  function timeoutRequests(IFunctionsRequest.Commitment[] calldata requestsToTimeoutByCommitment) external override {
```

Comment:
This is an awkward part about this change. Now to timeout requests the whole commitment has to be sent back and verified.

Reply:
Yeah, not great but I don't see a way around it.

Reply:
We are going to need to provide good support for this in the frontend.  I should probably be raised.